test/e2e/checkpoint_test.go

```diff
@@ -334,6 +334,10 @@ var _ = Describe("Podman checkpoint", func() {
 		IPBefore.WaitWithDefaultTimeout()
 		Expect(IPBefore.ExitCode()).To(Equal(0))
 
+		MACBefore := podmanTest.Podman([]string{"inspect", "-l", "--format={{.NetworkSettings.MacAddress}}"})
```

Comment:
Can we get a non-checkpoint test? Set `--mac-address ab:cd:ef:gh` and verify that it's set in the container?